docs/tutorials/02_ui/viz_tab.py

```diff
@@ -0,0 +1,213 @@
+"""
+========
+Tab UI
+========
+
+This example shows how to use the Tab UI. We will demonstrate how to
+create Tabs for:
+
+1. Slider controls for a Cube
+2. Checkboxes for Cylinder and Sphere
+3. Color combobox for Fury.
+
+First, some imports.
+"""
+from fury import ui, window, actor
+import numpy as np
+
+###############################################################################
+# First, we create the Tab UI.
+
+tab_ui = ui.TabUI(position=(49, 94), size=(300, 300), nb_tabs=3,
+                  draggable=True)
+
+###############################################################################
+# Slider Controls for a Cube for Tab Index 0
+# ==========================================
+#
+# Now we prepare content for the first tab.
+
+ring_slider = ui.RingSlider2D(initial_value=0,
+                              text_template="{angle:5.1f}°")
+
+line_slider_x = ui.LineSlider2D(initial_value=0,
+                                min_value=-10, max_value=10,
+                                orientation="horizontal",
+                                text_alignment="Top")
+
+line_slider_y = ui.LineSlider2D(initial_value=0,
+                                min_value=-10, max_value=10,
+                                orientation="vertical",
+                                text_alignment="Right")
+
+cube = actor.box(centers=np.array([[10, 0, 0]]),
+                 directions=np.array([[0, 1, 0]]),
+                 colors=np.array([[0, 0, 255]]),
```

Comment:
Can you rebase and use `1`instead of `255`

Reply:
@skoudoro Updated 😄